test/e2e/atlas/streams_with_clusters_test.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (atlas && streams_with_cluster)
```

Comment:
you need to add one like this
https://github.com/mongodb/mongodb-atlas-cli/blob/80d8b7fe783f372aba9b1b8ffa3b1cfba642e14a/build/ci/evergreen.yml#L840-L857
but with the right E2E_TAGS values and different  `["e2e","generic","atlas", "assigned_to_jira_team_cloudp_atlas_streams"] ` maybe `["e2e","streams","atlas", "assigned_to_jira_team_cloudp_atlas_streams"] and then a new build variant that won't run on PRs only on the mainline like

https://github.com/mongodb/mongodb-atlas-cli/blob/80d8b7fe783f372aba9b1b8ffa3b1cfba642e14a/build/ci/evergreen.yml#L1206-L1213

but updating `".e2e .atlas .kubernetes"` to `".e2e .atlas .streams" `

Reply:
Thank you. Updated in [9f339c8](https://github.com/mongodb/mongodb-atlas-cli/pull/2600/commits/9f339c8f193afb681f3a17e76903b4938f4995de)